src/app/task-list/task-list-item/task-list-item.component.ts

```diff
@@ -1,5 +1,6 @@
-import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { Component, OnInit, Input, Output, EventEmitter, TemplateRef } from '@angular/core';
 import { TaskItem } from '../../models/TaskItem';
+import { BsModalRef, BsModalService } from '../../../../node_modules/ngx-bootstrap';
```

Comment:
Please use it like  from 'ngx-bootstrap'; Avoid using import like this '../../../../node_modules/

Reply:
Used